docs/atomic/particle_class.rst

```diff
@@ -63,7 +64,7 @@ Some of these properties are returned as a `~astropy.units.Quantity` in
 SI units.
 
 >>> alpha.charge
-<Quantity 3.20435324e-19 C>
+<Quantity 3.204353242e-19 C>
```

Comment:
Interestingly, I'm getting the previous `<Quantity 3.20435324e-19 C>` version, without 2 as the last digit. Perhaps you're running an older version of astropy (3.2.1) or numpy (1.17.2 here)?